src/Aquarius.ONE.ClientSDK/Operations/Sample/SampleApi.cs

```diff
@@ -28,6 +29,34 @@ public class SampleApi
             _throwAPIErrors = throwAPIErrors;
         }
 
+        /// <summary>
+        /// Retrieves activities and their children based on criteria.
+        /// </summary>
+        /// <param name="authTwinRefId">
+        /// Optional: Filter results based on activities associated with a specific twin, defaults
+        /// to tenantId in token if not provided.
+        /// </param>
+        /// <param name="activityTypeId">
+        /// Optional: Defaults to all. Filters results to a specific ActivityType.
+        /// </param>
+        /// <param name="statusCode">
+        /// Optional: Defaults to all. Filters results to a specific status code.
+        /// </param>
+        /// <param name="priorityCode">
+        /// Optional: Defaults to all. Filters results to a specific priority code.
+        /// </param>
+        /// <param name="startDate">
+        /// Optional: Defaults to null. Filters results to those that start at or later than the
+        /// specified time. Supplying a StartDate without an EndDate is acceptable.
+        /// </param>
+        /// <param name="endDate">
+        /// Optional: Defaults to null. Filters results to those that end at or before than the
+        /// specified time. Supplying a EndDate without an StartDate is acceptable.
+        /// </param>
+        /// <param name="scheduleId">
+        /// Optional: Defaults to null. Filter results to those that are for this particular schedule.
+        /// </param>
+        /// <returns>One or more Activities that meet the criteria.</returns>
         public async Task<List<Activity>> GetActivitiesAsync(string authTwinRefId,

```

Comment:
What's the difference between this and the ActivityApi section of the SDK?  I think it may be confusing to the user.  Perhaps when getting a sample there should be an option to get related activities?  Or perhaps there could be methods like "GetSampleActivities(Guid operationId, ...)".  I'm trying to brainstorm some ways to simplify this

Reply:
There is no difference. This is a passthrough and calls GetActivitiesAsync. This was already in place and not part of the story under review here. I just added the comments to improve our docs.

But I believe that the functions parallel the functions in the MSApps where the Sample app has endpoints that just pass through to the Activity and Schedule MSApps and allow the consumer to make all of the calls needed using just the SampleSDK/SampleApi.

I'm open to any improvements and can make some changes as part of this story but I don't want to expand the scope too much :)